components/library/poppler/Makefile

```diff
@@ -62,18 +61,21 @@ CMAKE_OPTIONS += -DENABLE_BOOST=OFF
 # We don't have gpgmepp.
 CMAKE_OPTIONS += -DENABLE_GPGME=OFF
 # jpeg-turbo is not detected without this.
-CMAKE_OPTIONS += -DJPEG_LIBRARY=$(JPEG_LIBDIR)/libjpeg.so
 CMAKE_OPTIONS += -DENABLE_QT6=OFF
 CMAKE_OPTIONS += -DQt5Core_DIR=$(QT5_LIBDIR)/cmake/Qt5Core
 CMAKE_OPTIONS += -DQt5Gui_DIR=$(QT5_LIBDIR)/cmake/Qt5Gui
 CMAKE_OPTIONS += -DQt5Xml_DIR=$(QT5_LIBDIR)/cmake/Qt5Xml
 CMAKE_OPTIONS += -DQt5Widgets_DIR=$(QT5_LIBDIR)/cmake/Qt5Widgets
 CMAKE_OPTIONS += -DQt5Test_DIR=$(QT5_LIBDIR)/cmake/Qt5Test
+CMAKE_OPTIONS += -DTESTDATADIR=$(COMPONENT_DIR)/$(COMPONENT_NAME_1)-$(GIT_COMMIT_ID_1)
+
+# Note that most tests are brokenon OpenIndiana
```

Comment:
brokenon?

Reply:
BTW, how this comment aligns with the `results-all.master` with the following results?

```
100% tests passed, 0 tests failed out of 65
```